src/graph.py

```diff
@@ -635,6 +644,54 @@ def grid_search(self, prize_file, Gs, Bs, Ws):
         return forest, augmented_forest, params_by_nodes
 
 
+    def _eval_randomizations(self, params):
+        """
+        Convenience method which sets parameters and performs PCSF
```

Comment:
This doesn't seem like a suitable docstring. Doers it perform PCSF or perform randomizations? What is `params`?

Reply:
2540119b3d00fa9be87242c0ac50189ef788e1e3